docs/hello_nextflow/01_orientation.md

```diff
@@ -1,41 +1,33 @@
 # Orientation
 
-The Gitpod environment contains some test data that will be used in this workshop. All software required are already installed and configured in it too.
+The Gitpod environment contains some test data that will be used in this training course. All software required is already installed and configured in it too.
 
 !!! note
 
-    Follow [this link](../../envsetup/) if you have not yet setup your Gitpod environment.
+    Follow [this link](../../envsetup/) if you have not yet set up your Gitpod environment.
 
-## Getting started
+## Materials provided
 
-You will complete this module in the `hello-nextflow/` folder.
+Throughout this training course, we'll be working in the `hello-nextflow/` directory.
 
 ```bash
 cd /workspace/gitpod/hello-nextflow
 ```
 
-In this folder you will all test data, code and accessory needed to work through this training module.
+This directory contains all the code files, test data and accessory files you will need. Feel free to explore the contents of this directory; the easiest way to see what it contains is the use the `tree` command (here we generate a table of contents to the second level down).
```

Comment:
I'd not say it's the easiest way, I'd reckon the file browser within VS code might be easier, but tree is pretty good:
```suggestion
This directory contains all the code files, test data and accessory files you will need. Feel free to explore the contents of this directory; an easy way to see what it contains is the use the `tree` command (here we generate a table of contents to the second level down).
```